src/utils/openwhisk.js

```diff
@@ -66,6 +66,7 @@ function extractClientRequest(action) {
     selector: request.params.selector || '',
     url: `${reqPath}${query}`,
     rootPath: request.params.rootPath,
+    queryString: request.params.params,
```

Comment:
according to the schema example and the [spec](https://url.spec.whatwg.org/#dom-url-search), the `queryString` property should either be empty or start with a `?`.